[2024-10-03T22:44:01.673-0400] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T22:44:16.001-0400] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: huggingface_pdf_pipeline.upload_pdfs_task manual__2024-10-04T02:38:59.076013+00:00 [queued]>
[2024-10-03T22:44:16.276-0400] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: huggingface_pdf_pipeline.upload_pdfs_task manual__2024-10-04T02:38:59.076013+00:00 [queued]>
[2024-10-03T22:44:16.278-0400] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T22:44:16.567-0400] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): upload_pdfs_task> on 2024-10-04 02:38:59.076013+00:00
[2024-10-03T22:44:16.571-0400] {standard_task_runner.py:72} INFO - Started process 32803 to run task
[2024-10-03T22:44:16.573-0400] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'huggingface_pdf_pipeline', 'upload_pdfs_task', 'manual__2024-10-04T02:38:59.076013+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/huggingface_pdf_airflow_dag.py', '--cfg-path', '/tmp/tmp5w1bcsm1']
[2024-10-03T22:44:16.576-0400] {standard_task_runner.py:105} INFO - Job 40: Subtask upload_pdfs_task
[2024-10-03T22:44:39.156-0400] {job.py:229} INFO - Heartbeat recovered after 47.08 seconds
[2024-10-03T22:44:54.664-0400] {task_command.py:467} INFO - Running <TaskInstance: huggingface_pdf_pipeline.upload_pdfs_task manual__2024-10-04T02:38:59.076013+00:00 [running]> on host LAPTOP-GD5THE3N.
[2024-10-03T22:44:57.975-0400] {job.py:229} INFO - Heartbeat recovered after 18.94 seconds
[2024-10-03T22:45:10.511-0400] {job.py:229} INFO - Heartbeat recovered after 12.65 seconds
[2024-10-03T22:45:11.771-0400] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='huggingface_pdf_pipeline' AIRFLOW_CTX_TASK_ID='upload_pdfs_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-04T02:38:59.076013+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-04T02:38:59.076013+00:00'
[2024-10-03T22:45:11.774-0400] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-03T22:45:11.775-0400] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-03T22:45:11.777-0400] {logging_mixin.py:190} INFO - Current task name:upload_pdfs_task state:running start_date:2024-10-04 02:44:16.083217+00:00
[2024-10-03T22:45:11.778-0400] {logging_mixin.py:190} INFO - Dag name:huggingface_pdf_pipeline and current dag run status:running
[2024-10-03T22:45:11.779-0400] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T22:45:11.780-0400] {logging_mixin.py:190} INFO - Error uploading / to Azure Blob Storage Container 1: Please specify a container name and blob name.
[2024-10-03T22:45:11.782-0400] {logging_mixin.py:190} INFO - Error uploading h to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'h'
[2024-10-03T22:45:11.784-0400] {logging_mixin.py:190} INFO - Error uploading o to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'o'
[2024-10-03T22:45:11.799-0400] {logging_mixin.py:190} INFO - Error uploading m to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'm'
[2024-10-03T22:45:11.812-0400] {logging_mixin.py:190} INFO - Error uploading e to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'e'
[2024-10-03T22:45:11.814-0400] {logging_mixin.py:190} INFO - Error uploading / to Azure Blob Storage Container 1: Please specify a container name and blob name.
[2024-10-03T22:45:11.816-0400] {logging_mixin.py:190} INFO - Error uploading p to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'p'
[2024-10-03T22:45:11.819-0400] {logging_mixin.py:190} INFO - Error uploading a to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'a'
[2024-10-03T22:45:11.822-0400] {logging_mixin.py:190} INFO - Error uploading t to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 't'
[2024-10-03T22:45:11.824-0400] {logging_mixin.py:190} INFO - Error uploading r to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'r'
[2024-10-03T22:45:11.826-0400] {logging_mixin.py:190} INFO - Error uploading i to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'i'
[2024-10-03T22:45:11.828-0400] {logging_mixin.py:190} INFO - Error uploading c to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'c'
[2024-10-03T22:45:11.830-0400] {logging_mixin.py:190} INFO - Error uploading k to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'k'
[2024-10-03T22:45:11.831-0400] {logging_mixin.py:190} INFO - Error uploading / to Azure Blob Storage Container 1: Please specify a container name and blob name.
[2024-10-03T22:45:11.832-0400] {logging_mixin.py:190} INFO - Error uploading D to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'D'
[2024-10-03T22:45:11.833-0400] {logging_mixin.py:190} INFO - Error uploading o to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'o'
[2024-10-03T22:45:11.834-0400] {logging_mixin.py:190} INFO - Error uploading w to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'w'
[2024-10-03T22:45:11.835-0400] {logging_mixin.py:190} INFO - Error uploading n to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'n'
[2024-10-03T22:45:11.836-0400] {logging_mixin.py:190} INFO - Error uploading l to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'l'
[2024-10-03T22:45:11.837-0400] {logging_mixin.py:190} INFO - Error uploading o to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'o'
[2024-10-03T22:45:11.838-0400] {logging_mixin.py:190} INFO - Error uploading a to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'a'
[2024-10-03T22:45:11.838-0400] {logging_mixin.py:190} INFO - Error uploading d to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 'd'
[2024-10-03T22:45:11.839-0400] {logging_mixin.py:190} INFO - Error uploading s to Azure Blob Storage Container 1: [Errno 2] No such file or directory: 's'
[2024-10-03T22:45:11.840-0400] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T22:45:17.342-0400] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T22:45:17.345-0400] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=huggingface_pdf_pipeline, task_id=upload_pdfs_task, run_id=manual__2024-10-04T02:38:59.076013+00:00, execution_date=20241004T023859, start_date=20241004T024416, end_date=20241004T024517
[2024-10-03T22:45:17.530-0400] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-03T22:45:17.532-0400] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-03T22:45:17.534-0400] {logging_mixin.py:190} INFO - Dag name:huggingface_pdf_pipeline queued_at:2024-10-04 02:38:59.673155+00:00
[2024-10-03T22:45:17.535-0400] {logging_mixin.py:190} INFO - Task hostname:LAPTOP-GD5THE3N. operator:PythonOperator
[2024-10-03T22:45:17.562-0400] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T22:45:20.310-0400] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T22:45:20.396-0400] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-03T22:45:21.645-0400] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
